docs/product-model.md

```diff
@@ -36,854 +38,908 @@ FpML, the CDM includes defined Synonyms that map to FpML (and other
 models) and can be used for transformation purposes. More details on
 Synonyms are provided in the Mapping (Synonym) section of this document.
 
-## TradableProduct {#tradable-product}
-
-A tradable product represents a financial product that is ready to be
-traded, meaning that there is an agreed financial product, price,
-quantity, and other details necessary to complete an execution of a
-security or a negotiated contract between two counterparties. Tradable
-products are represented by the `TradableProduct` type.
-
-``` Haskell
-type TradableProduct:
-   product Product (1..1)
-   tradeLot TradeLot (1..*)
-   counterparty Counterparty (2..2)
-   ancillaryParty AncillaryParty (0..*)
-   adjustment NotionalAdjustmentEnum (0..1)
-```
+## Financial Products {#product}
 
----
-**Note:**
-The conditions for this data type are excluded from the snippet above
-for purposes of brevity.
+A financial product is an instrument that is used to transfer financial
+risk between two parties. 
 
----
+### Asset  {#asset}
 
-The primary set of attributes represented in the `TradableProduct` data
-type are ones that are shared by all trades and transactions. For
-example, every trade has a price, a quantity (treated jointly as a trade
-lot), and a pair of counterparties. In some cases, there are ancillary
-parties, or an allowable adjustment to the notional quantity. All of the
-other attributes required to describe a product are defined in distinct
-product data types.
+The concept of "Asset" has been introduced in the CDM in version 6.0 to improve the modelling of simpler financial products
+and to continue the expansion of the CDM from beyond its heritage focus on over the counter derivatives.
```

Comment:
Uncomfortable with these 2 lines that talk about the model's "evolution", "improvement" and "heritage". The documentation should only describe the model "as it is" (in the current released version), not how it was or how it should be.

I note that you could remove those 2 lines and the documentation still stands on its own two feet by describing what an asset is.

Reply:
understood - P3 will be final version